Competition2018/src/main/java/competition/subsystems/elevator/commands/TalonCurrentMonitor.java

```diff
@@ -0,0 +1,41 @@
+package competition.subsystems.elevator.commands;
+
+import java.util.ArrayDeque;
+import xbot.common.controls.actuators.XCANTalon;
+
+public class TalonCurrentMonitor {
+
+    XCANTalon talon;
+    final int CURRENT_AVERAGING_WINDOW;
```

Comment:
Set the value here rather than in the constructor

Reply:
okay!